src/test/java/org/opensearch/neuralsearch/processor/rerank/ByFieldRerankProcessorTests.java

```diff
@@ -985,7 +985,10 @@ public void testRerank_throwsExceptionOnHavingNonNumericValue_WhenTargetFieldHas
         ArgumentCaptor<Exception> argumentCaptor = ArgumentCaptor.forClass(Exception.class);
         verify(listener, times(1)).onFailure(argumentCaptor.capture());
 
-        assertEquals("The field mapping to rerank by [hello world] is not Numerical", argumentCaptor.getValue().getMessage());
+        assertEquals(
+            "The field mapping to rerank by [hello world] is not Numerical, instead of type [java.lang.String]",
```

Comment:
Can you turn it into a string format, where `[java.lang.String]` gets in the same way as `typeOfMapping`?